packages/visualizations/src/components/Map/WebGl/ChoroplethVectorTiles.svelte

```diff
@@ -1,5 +1,3 @@
-<svelte:options immutable={true} />
```

Comment:
Why this change here and in MapRender? We had a look at this one year ago I think, and we opted for this because of our use of a global `options` parameter containing an object. Without `immutable`, Svelte will consider that the object will have changed at every render. Overall this only has drawbacks, and usually if removing it seems to fix something, it's probably a symptom of something broken coming from our own React code (typically mutating the `options` object instead of creating a new one).

For reference:
- [the equality check difference between the two modes is defined here](https://github.com/sveltejs/svelte/blob/master/src/runtime/internal/utils.ts)
- [the two functions](https://github.com/sveltejs/svelte/blob/master/src/runtime/internal/utils.ts) (the `a!= a` is there to detect `NaN`, the rest is rather clear)